addons/mrp/models/stock_warehouse.py

```diff
@@ -276,3 +276,15 @@ def _update_name_and_code(self, name=False, code=False):
             if warehouse.manufacture_pull_id and name:
                 warehouse.manufacture_pull_id.write({'name': warehouse.manufacture_pull_id.name.replace(warehouse.name, name, 1)})
         return res
+
+class Orderpoint(models.Model):
+    _inherit = "stock.warehouse.orderpoint"
+
+    @api.constrains('product_id')
+    def check_product_is_not_kit(self):
+        for orderpoint in self:
```

Comment:
Can you do it in batch to avoid any futur performance issue because of this ? (same comment for the other constraint)

Reply:
@ryv-odoo Code updated! I also changed the validation error messages. Note that doing this in batch will prevent the use of `_bom_find` in 13.0 since the method only works with one product:
```python
@api.model
def _bom_find_domain(self, product_tmpl=None, product=None, picking_type=None, company_id=False, bom_type=False):
    if product:
        if not product_tmpl:
            product_tmpl = product.product_tmpl_id
        domain = ['|', ('product_id', '=', product.id), '&', ('product_id', '=', False), ('product_tmpl_id', '=', product_tmpl.id)]
    elif product_tmpl:
        domain = [('product_tmpl_id', '=', product_tmpl.id)]
```